packages/php/email-editor/changelog/59859-wooplug-4979-integrate-the-html-to-text-library-into-the-email-editor-2

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Add licensing and security policy.
```

Comment:
No need for extra changelog entries; just update the first one accordingly.

Reply:
Since the 2 items are not strictly related, and could have been handled in 2 different PRs (one adding the security and basic license file, the other editing the license file to add the new extra copyright), it seems like having 2 changelog entries may be cleaner, and maybe interesting for consumers of the package?

Reply:
I see the point. Well, I don't have any good insight into the package consumers profile - feel free to ignore this comment =)